javascript-modal/main.js

```diff
@@ -0,0 +1,22 @@
+var isOn = true;
+var isOff = true;
+var modalButton = document.querySelector('.modal-button');
+var modalContainer = document.querySelector('.modal-container');
+
+function modalOpen() {
+  if (isOn) {
+    modalContainer.className = 'modal-container show';
+  }
+}
+
+modalButton.addEventListener('click', modalOpen);
+
+var noButton = document.querySelector('.no');
+
+function modalClose() {
+  if (isOff) {
```

Comment:
You're on the right track, but this is still unnecessary. If you have a boolean flag that never changes in value, that flag is unnecessary. Again, if somebody clicks the "Open Modal" button, they want to open the modal every time. If somebody clicks the "No" button, they want to close the modal every time.